index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
```

Comment:
Order for importing packages

1. packages (npm install, inbuilt packages)
2. custom modules (group the same folder imports)
3. Intializations (after a space)